### DISCUSSION TITLE
【2311260】英灵学院

### DISCUSSION THREAD

**Discussion by hisoio** [pjy612/SteamManifestCache#152] [Share]
appid:2311260
keys：
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13761866/default.zip)

2311261;2d2edc96be3759131e5e09b357a090b74d9416734d5f9d3f326cf6ce5adbc0c0

**Comment by pjy612** [comment]
感谢扩充库存

**Reply by pjy612** [reply]
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/6e19b47d-bead-4188-afca-7020494861af)

已验
